cmd/algorand-indexer/internal/ledgerTest.go

```diff
@@ -0,0 +1,100 @@
+package internal
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/algorand/go-algorand/ledger/ledgercore"
+
+	"github.com/algorand/indexer/fetcher"
+	"github.com/algorand/indexer/idb"
+	"github.com/algorand/indexer/processor/blockprocessor"
+	"github.com/algorand/indexer/util"
+)
+
+// LedgerTestCmd is the account validator utility.
+var LedgerTestCmd *cobra.Command
+
+type params struct {
+	indexerDataDir string
+	catchpoint     string
+	algodAddr      string
+	algodToken     string
+}
+
+func init() {
+	var (
+		config params
+	)
+
+	LedgerTestCmd = &cobra.Command{
+		Use:   "ledger-test",
+		Short: "ledger-test",
+		Long:  "Initialize the ledger without a database.",
+		Run: func(cmd *cobra.Command, _ []string) {
+			run(config)
+		},
+	}
+
+	LedgerTestCmd.Flags().StringVar(&config.indexerDataDir, "data-dir", "", "Indexer data directory.")
+	LedgerTestCmd.Flags().StringVar(&config.catchpoint, "catchpoint", "", "Catchpoint to use for initializing.")
+	LedgerTestCmd.Flags().StringVarP(&config.algodAddr, "algod-net", "", "", "host:port of algod")
+	LedgerTestCmd.Flags().StringVarP(&config.algodToken, "algod-token", "", "", "api access token for algod")
```

Comment:
nit: should these be StringVar too since they don't have shorthand code either.

Reply:
Thanks, this was probably from copy/pasting these flags from somewhere else.